### ISSUE TITLE
Can't package my problem

### ISSUE THREAD

**Issue by AmrElTaieb** [shoes/shoes4#1453]
I installed shoes4 and jruby and the jdk.
I made my first shoes app. It ran properly.
But when I type
  shoes package --jar C:\Users\Asd\Desktop\Shoes\Shoes.rb
It doesn't pack my app. It doesn't do anything actually. 
I tried a different command
  shoes package --windows C:\Users\Asd\Desktop\Shoes\Shoes.rb
and I got the same result.
I tried to run the package command from the directory where my .rb file is and that's what happened
  C:\Users\Asd>cd C:\Users\Asd\Desktop\Shoes\

  C:\Users\Asd\Desktop\Shoes>shoes package --windows C:\Users\Asd\Desktop\Shoes\Shoes.rb
  C:/Users/Asd/Desktop/Shoes/Shoes.rb:1:in `<main>': uninitialized constant Shoes (NameError)

When I open my cmd I typed
  ruby -version
Surprisingly I got this output
  ruby 2.2.6p396 (2016-11-15 revision 56800) [x64-mingw32]
  -e:1:in `<main>': undefined local variable or method `rsion' for main:Object (NameError)

I typed 
  shoes --version
and nothing happened like when I tried to pack my program.
What could be the problem ?

**Comment by MichaelDimmitt** [comment]
@AmrElTaieb, I package on my mac; So please take this advice with a grain of salt. 

However, consider trying ```bin/shoes package --jar C:\Users\Asd\Desktop\Shoes\Shoes.rb```
<b>added "bin"</b> to the beginning of your command.
Cheers, 😄 . If successful let me know. 
If not I am sure others will chime in by end of day or next few days. 😄

**Comment by MichaelDimmitt** [comment]
additionally try moving shoes4 to: (a hunch)
C:\Users\Asd\Shoes4

**Comment by AmrElTaieb** [comment]
@MichaelDimmitt 
I added bin to the beginning of my command.
Unfortunately it didn't work, but thanks for the advice 😄 

C:\Users\Asd>bin/shoes package --jar C:\Users\Asd\Desktop\Shoes\Shoes.rb
'bin' is not recognized as an internal or external command, operable program or batch file.

**Comment by MichaelDimmitt** [comment]
@AmrElTaieb, I think these two details would provide useful information:
What is the path where shoes4 is located?
What is the path of the project you are trying to package?

**Comment by ccoupe** [comment]
Including @MichaelDimmitt  suggestions. You need to  change the name and path to your app. C:\Users\Asd\Desktop\Shoes\Shoes.rb is not a path that Shoes 3 or Shoe4 can navigate to.  Desktop just won't do what you want. Point to where your app is really stored and don't name it Shoes.rb (we have one those so you invite confustiom

**Comment by AmrElTaieb** [comment]
@MichaelDimmitt surprisingly I found shoes4 in this directory:
C:\Ruby22-x64\lib\ruby\gems\2.2.0\gems\shoes-4.0.0.pre10
It should've been in a jruby directory shouldn't it ?
@ccoupe As for my project I changed its name and path as you suggested:
D:\Gui\gui.rb

**Comment by PragTob** [comment]
@AmrElTaieb yes it should be in the JRuby directory. It might be that the `gem` executable still belongs to CRuby. JRuby comes with `jgem` to use it alongside another ruby installation, however it might be better to use a ruby version switcher although I'm not familiar with the Windows landscape too well to know which work
